### DISCUSSION TITLE
[@types/node] error TS2614: Module '"node:test"' has no exported member 'beforeEach'

### DISCUSSION THREAD

**Discussion by jtuchel** [DefinitelyTyped/DefinitelyTyped#62978] [Issues with a @types package] [answered]
The video **What's new in the native Node.js Test Runner 2022** from **Erick Wendel** shows that it's possible to import  `beforeEach` from the package `node:test` => https://youtu.be/xU1EmrhQCfw?t=512

I'm using Node 19 and it works with JS but when using TS with @types/node v18.11.4 I get the following error

> error TS2614: Module '"node:test"' has no exported member 'beforeEach'

I don't know if this is a Node 19 feature but it would be nice if I could import this function without getting a compile error :)

[a comment by an automated account is not shown]

**Comment by peterblazejewicz** [comment, marked as answer]
@jtuchel the support depends on this pr status: #62274
